BayPass/Model/Map/MapAnnotation.swift

```diff
@@ -16,12 +16,16 @@ class MapAnnotation: NSObject, MKAnnotation {
     var coordinate: CLLocationCoordinate2D
     var icon: UIImage
     var color: UIColor
+    var latitude: CLLocationDegrees
```

Comment:
Why did you add these variables here?

Reply:
I didn't add these here, these are from master branch